.typos.toml

```diff
@@ -1,38 +1,54 @@
 [default]
-extend-ignore-re = ["[a-zA-Z0-9]{44}", "[a-zA-Z0-9]{9}"]
+extend-ignore-identifiers-re = [
+  # Short git hashes
+  "^[a-f0-9]{7}$",
+  # Digests
+  "^[a-zA-Z0-9]{43}$",
+  # Identifiers
+  "^[a-zA-Z0-9]{44}$",
+  # Base64
+  "^[a-zA-Z0-9_]{47}$",
+  # Base64
+  "^[a-zA-Z0-9]{49}$",
+  # Long keys and such
+  "^[a-zA-Z0-9]{56,}$",
+]
+# Truncated hex IDs (ex. "0x9...a9b")
+extend-ignore-re = ["0x[a-f0-9]\\.{3}[a-f0-9]{3}"]
+
+[type.hex-move]
+extend-glob = ["hex.move"]
+extend-words = { ba = "ba" }
+
+[type.pg]
+extend-glob = ["pg.rs"]
+extend-words = { ba = "ba" }
 
 [default.extend-words]
-# accepted plurals
-froms = "froms"
-alls = "alls"
-somes = "somes"
 # ordinal numbers (2nd, ...)
 nd = "nd"
 # random identifiers, variables, functions, ...
-bttr = "bttr"
-dbe = "dbe"
-thr = "thr"
-fo = "fo"
-BA = "BA"
-DAA = "DAA"
-UE = "UE"
 tto = "tto"
```

Comment:
What's the difference between `extend-words` and `extend-identifiers`? Shouldn't things like that be identifiers instead?